src/renderer/components/+workloads-replicasets/replicasets.tsx

```diff
@@ -78,15 +75,3 @@ export class ReplicaSets extends React.Component<Props> {
   }
 }
 
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency